sdcm/nemesis.py

```diff
@@ -539,7 +540,12 @@ def _add_and_init_new_cluster_node(self, old_node_ip=None, timeout=HOUR_IN_SEC *
         return new_node
 
     def _terminate_cluster_node(self, node):
-        self.cluster.terminate_node(node)
+        if isinstance(self.cluster, GkeScyllaPodCluster):
+            terminate_method_name = random.choice(('terminate_k8s_host', 'terminate_k8s_node', 'terminate_node'))
```

Comment:
allow terminate_node for both GKE and Minikube, raise for all other methods

Reply:
done